src/database/db.ts

```diff
@@ -2,6 +2,7 @@
 import { Sequelize } from 'sequelize';
 import getDatabaseConfig from '../config/config';
 import Models from './models';
+import logger from '../utils/logger';
```

Comment:
can you make folder structure like this
- if it's utils create new file index where it's will export every file from utils folder then all you will need to do is to import ../utils only that's enaught and simple

Reply:
Resolved